spec/controllers/projects_controller_spec.rb

```diff
@@ -250,4 +250,73 @@
       end
     end
   end
+
+
+  describe "online_days" do
+    context "when has a value greater than 60" do
+      let(:online_days_error_message) {
+        I18n.t('activerecord.attributes.project.online_days') + ' ' + \
+        I18n.t('activerecord.errors.models.project.attributes.online_days.less_than_or_equal_to')
+      }
+
+      before(:each) do
+        user_decorated = UserDecorator.new(user)
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_decorated)
+      end
+
+      context "when the request is a POST" do
+        before do
+          post :create, { locale: :pt, project: project.attributes }
+        end
+
+        context "when the current_user is a juntos' admin" do
+          let(:user) { create(:user, admin: true) }
+          let(:project) { build(:project, permalink: 'available_permalink', online_days: 61) }
+
+          it "should not show any error message" do
+            expect(flash[:alert]).to be_nil
+          end
+        end
+
+        context "when the current_user is a normal user" do
+          let(:user) { create(:user, admin: false) }
+          let(:project) { build(:project, permalink: 'other_available_permalink', online_days: 61) }
+
+          it 'should return a flash error message for the online_days field' do
```

Comment:
in the other tests you're using double quotes. It'd be nice to keep a pattern here ;) 💅